main.go

```diff
@@ -171,12 +172,18 @@ func registerDynamicCapabilities(port string, log logger.Handler) {
 
 }
 func registerWorkloads(port string, log logger.Handler) {
-	release, err := config.GetLatestReleases(1)
-	if err != nil {
-		log.Info("Could not get latest stable release")
+	// release, err := config.GetLatestReleases(1)
+	// if err != nil {
+	// 	log.Info("Could not get latest stable release")
+	// 	return
+	// }
+	// version := release[0].TagName
+	version := "1.11.4"
```

Comment:
Why is this hardcoded?

Reply:
Ooops, was testing something and it got committed by mistake, changing it